public/static/locales/en/enrolment.json

```diff
@@ -9,6 +52,20 @@
     "noSelectedOccurrences": "No selected event times"
   },
   "pageTitle": "Enrol for the event",
+  "studyLevel": {
```

Comment:
Would usage of https://www.i18next.com/translation-function/plurals#interval-plurals avoid some repetition here?

Reply:
Started to use i18next-intervalplural-postprocessor here. Needed some configuration and new logic when creating grade level options.